ci/pipelines/deploy-to-test.yml

```diff
@@ -709,6 +723,9 @@ resource_types:
       tag: latest
 
 groups:
+  - name: stream-s3-sqs
```

Comment:
Please can you move this to later in the list of groups, this would make it the first thing in the headings in the deploy-to-test pipeline (since the UI is ordered the same as this groups list).

Currently the order is:

1. all apps alphabetically
2. all supporting things mostly-alphabetically (notifications is in the wrong place sitting between telegraf and update-deploy-to-test-pipeline)

So if you could move this to between nginx-forward-proxy and telegraf that would be good 👍

Reply:
[Done](https://github.com/alphagov/pay-ci/pull/645/commits/fb05adf865e241139ced0650d9bc0bbd75dcf93b). Looks like it works: https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/deploy-to-test/jobs/build-and-push-stream-s3-sqs-to-test-ecr/builds/1.